src/utils.js

```diff
@@ -72,7 +73,7 @@ const memoize1 = (f) => (arg) => {
 };
 
 const reg = (i) => new RegExp(`\\$\\{([\\s]+?|\\s?)${i}([\\s]+?|\\s?)}`);
```

Comment:
The rail graph of this regexp looks like (assume `i` is `"XXXX"`):
https://regexper.com/#%5C%24%5C%7B(%5B%5Cs%5D%2B%3F%7C%5Cs%3F)XXXX(%5B%5Cs%5D%2B%3F%7C%5Cs%3F)%7D

The capturing group is not used, and `\s+|\s?` and `\s*` should be equivalent. I've tried
```javascript
const reg = (i) => new RegExp(`\\$\\{\\s*${i}\\s*}`);
```
And it does not break any of the existing test cases.
Shall we simplify this regexp as well?